test/unit/HTMLBuilder.test.ts

```diff
@@ -152,4 +153,46 @@ describe('HTMLGenerator', () => {
       generateHTML([elm], badStatementResults, [], 'test');
     }).toThrowError();
   });
+  test('clause coverage percent with a function ignores function in calculation', () => {
```

Comment:
I would just add a space above this test to be consistent with the rest of the file :)